### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by rhoggSugarcrm**
Make it clearer that Thorn is meant to be used with Mocha.

### CODE REVIEW

File: Docs.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ setting up a SugarCRM's REST API testing environment and interacting with it.
...
+
+Thorn tests are run using the Mocha test runner. To add Thorn as a dependency, run
+
+```bash
+$ yarn add --dev '@sugarcrm/thorn'
+```
+
+You will need to invoke Mocha in order to run Thorn. Check out
+[Mocha's usage information][mocha-usage], [grunt-mocha], and
+[gulp-spawn-mocha] for information about instantiating Mocha.
```
Comment (rhoggSugarcrm) [inline Docs.md]: I'm not providing Sugar-specific information here, just a general comment about how to invoke Thorn tests.

File: README.md [hunk 2]
```diff
@@ -2,12 +2,6 @@
 
 Thorn is composed of a set of helper abstractions built to ease the process of setting up a [SugarCRM](https://www.sugarcrm.com) instance's REST API testing environment and interacting with it.
 
-## Installation
```
Comment (jcsmorais) [inline README.md]: @rhoggSugarcrm why not keeping at least a reference to installation steps here?

Comment (rhoggSugarcrm) [inline README.md]: TBH, it makes sense to put it back. I think I was considering being more explicit about how Sugar uses Thorn internally when I originally took this out.

Comment (rhoggSugarcrm) [reply README.md]: Restored!

**Pull request merged**